tutorials/themes/tutorials-theme/navbar.html

```diff
@@ -39,9 +39,10 @@ <h2 class="card-title text-center"><a href="{{ pathto('examples') }}" class="tex
 </div>
 
 {% else %}
-
 <nav class="navbar navbar-expand-lg bg-navbar">
-  <a class="navbar-brand" href="/astropy-tutorials"style="padding: 0px 5px;"><img src="{{ pathto('_static/' + 'astropy_logo.png') }}" style="height:38px;"></a>
+  <a class="navbar-brand" href="http://www.astropy.org/astropy-tutorials/" style="padding: 0px 5px;">
```

Comment:
We don't want to assume the domain here, so change to `href="/astropy-tutorials"`